docs/release/trg-0/trg-1-09.md

```diff
@@ -0,0 +1,43 @@
+---
+title: TRG 1.09 - UX consistency Style Guideline for UI
```

Comment:
Hi @RolaH1t  you can directly share a suggestion for the code within this pr like: 
```suggestion
title: TRG 1.09 - UX consistency / Styleguide for UI
```